growth/too/tasks/tiles.py

```diff
@@ -240,9 +241,10 @@ def gen_structs(params):
         raise ValueError(
             'Need tilesType to be moc, greedy, hierarchical, galaxy or ranked')
 
-    coverage_struct = gwemopt.coverage.timeallocation(params,
-                                                      map_struct,
-                                                      tile_structs)
+    tile_structs, coverage_struct =\
```

Comment:
Please add missing space after binary operand.

Reply:
flake8 --show-source growth passes for me?

Reply:
Nonetheless, please add a space.

Reply:
I cannot make flake8 pass with this.